lib/NewAnimator.js

```diff
@@ -1,16 +1,14 @@
 require('enyo');
 
 var
-	kind = require('./kind'),
 	utils = require('./utils');
 
 var
 	CoreObject = require('./CoreObject'),
 	Loop = require('./Loop');
 
-module.exports = kind.singleton({
+module.exports = CoreObject({
```

Comment:
Ya. Discovered that I need to rework this PR anyway as you can't extend existing methods with this approach ... you just overwrite them.